packages/federated-identity-service/package.json

```
@@ -43,6 +43,7 @@
     "@twake/config-parser": "*",
     "@twake/crypto": "*",
     "@twake/logger": "*",
+    "@twake/utils": "*",
```

Biome (2.1.2)

[error] The key @twake/utils was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)